README.md

```diff
@@ -385,6 +386,53 @@ config := hazelcast.NewConfig()
 config.NetworkConfig().AddAddress("some-ip-address:port")
 hazelcast.NewClientWithConfig(config)
 ```
+
+### 1.4.3. Hazelcast Properties
+
+Hazelcast property is a property which can be set programmatically through `config.SetProperty` or with an environment variable.
+The value of this property will be:
+
+* the programmatically configured value, if programmatically set,
+* the environment variable value, if the environment variable is set,
+* the default value, if none of above is set.
+
+For example: 
+
+```go
+InvocationTimeoutSeconds = NewHazelcastPropertyInt64WithTimeUnit("hazelcast.client.invocation.timeout.seconds",
+    120, time.Second)
+```
+
+
+as we see `invocation timeout` is `120 seconds` by default. You can change this as follows:
```

Comment:
The above property specifies the timeout duration to give up the invocations when a member in the member list is not reachable, and its default value is `120` seconds. You can change this value programmatically or using an environment variable, as shown below.